pkg/cmd/run/download/download_test.go

```diff
@@ -186,7 +186,7 @@ func (f *fakePlatform) List(runID string) ([]shared.Artifact, error) {
 	return artifacts, nil
 }
 
-func (f *fakePlatform) Download(url string, dir safepaths.Absolute) error {
+func (f *fakePlatform) Download(url string, dir safepaths.Absolute, force bool, skip bool) error {
```

Comment:
Though `platform` is different than `destinationWriter` in `gh release download`, I'm looking at the various clobber and skip tests to see what makes sense for `gh run download` tests.

https://github.com/cli/cli/blob/0acd40418a7e32e72f373d8b846a7bfadc110cfb/pkg/cmd/release/download/download_test.go#L401-L555

Again, still grokking the current code, but the `gh run download` code just assumes that anything other than an error means a file was downloaded.  However, if the user wants to skip previously downloaded files, then I think there is more that has to change here. 🤔